src/main/resources/fxml/tokenInput.fxml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<BorderPane prefHeight="400.0" prefWidth="600.0" styleClass="header-section" xmlns="http://javafx.com/javafx/17.0.2-ea" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.io.gittracker.controllers.TokenInput">
+    <center>
+        <VBox>
+            <Label text="Type your github token" />
```

Comment:
Labelka może nie mieścić się na ekranie, powinna zajmować szerokość całego okna
Przykład poniżej:
![Zrzut ekranu z 2024-03-12 23-56-44](https://github.com/DavidJones1102/Git-Tracker/assets/57900677/4716c47a-0cc0-4d8c-ac47-825755c94885)

Reply:
I cannot reproduce issue but It should work now